helm/idol-answerserver/values.yaml

```
@@ -31,6 +31,17 @@ idolImage:
   # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}
   repo: "answerserver"
 
+# answerServerVolumeSize -- Size of the PersistentVolumeClaim that is created
+# for each AnswerServer instance. The Kubernetes cluster will need to provide
+# enough PersistentVolumes to satisify the claims made for the desired number
+# of AnswerServer instances.
+answerServerVolumeSize: 16Gi
```

```suggestion
answerServerVolumeSize: 2Gi
```
`16Gi` seems excessive for answerserver. Even `2Gi` is a lot for licensing, but if we allow model cache data to go here then 2Gi is probably reasonable.